### DISCUSSION TITLE
리드잇 잡지 원고 모집 글

### DISCUSSION THREAD

**Discussion by TaeHyoungKwon** [ThinkAboutSoftware/AcademicConference#203] [General]
https://best-bag-0db.notion.site/zine-5-8c1869153bf74114adb60a4dd584b549

![image](https://user-images.githubusercontent.com/18538418/164079052-5a5887c8-0394-4061-9e5c-8a7598c55ecf.png)

리드잇 이라는 잡지에서 원고 모집 글이 올라온 걸 우연히 보았습니다 뽑히면 잡지 발행될 때마다 컬럼을 쓸 수 있는 기회도 주는 것 같습니다 ㅎㅎ - 이번 주제는 '개발자의 일 (Who am I : Developer)’ 인데.. 마침 책과 관련된 얘기도 많이 나와서 저흰 쓸 것도 많을 것 같아요 ㅋㅋㅋ

5월 15일 까지네요~ (저는 한번 지원해보려합니다 😄 )

**Comment by jongfeel** [comment]
아니?? 새벽 4시에? ㅋㅋㅋㅋ
무슨 일 있으신 건 아니죠?

링크 가봤는데 저도 한번 해보고 싶다는 생각이 드네요
리드잇진이라는 잡지가 있다는 것도 처음 알았습니다.

1호를 잠깐 봤는데 임백준님도 있고 기획해서 만든 인터뷰 잡지 같군요.
1호부터 쭉 읽을거리가 생겨서 즐겁습니다 ㅎㅎㅎ

저도 한번 해보겠습니다.
되면 좋겠군요
